update-colvars-code.sh

```diff
@@ -558,7 +567,7 @@ then
   echo ""
 
   # Apply patch for Gromacs files
-  patch ${patch_opts} -d ${target} < ${source}/gromacs/gromacs-${GMX_VERSION}.patch
+  # patch ${patch_opts} -d ${target} < ${source}/gromacs/gromacs-${GMX_VERSION}.patch
```

Comment:
Maybe you forget to apply the GROMACS patch?

Reply:
😩

Reply:
thanks!